### DISCUSSION TITLE
In single-player games, diving suits disappear.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13926] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

In single-player games, diving suits disappear.

The latest patch made the items return to the boat after the end of the round, but the suits continue to disappear.
Today, diving suits disappeared 2 times.
For the first time, 2 combat suits disappeared.
At the second time the slip suit has gone missing.
Please, clarify the issue.
Items left at the station by bots are returned to the submarine upon undocking.

Are items left at the station by the player returned to the submarine?

Are items left at the station returned to the submarine if, instead of undocking, I save the game and exit?

I had a mission with passengers and (to save energy) the oxygen generator was turned off.
Passengers usually wear their own breathing masks.
But this time the passenger wore a slip suit because before the docking there was an attack of raptors and partial flooding of the boat. I suppose that at the end of the round the passenger has gone with my slip suit.

Question: If there was a mission with passengers, and at the time of docking the passenger was in a diving suit, will the diving suit return to the boat or disappear forever along with the passenger?

### Reproduction steps

1. get passenger transport mission
2. force the passenger to wear a diving suit
3. dock 

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
You have reported this issue several times already. [We've implemented a fix for it](https://github.com/FakeFishGames/Barotrauma/discussions/13652), but it unfortunately hasn't been included in the public build yet.
